From: Andrew Morton <akpm@linux-foundation.org>
Subject: mmpage_owner-filter-out-stacks-by-a-threshold-counter-fix
Date: Thu May 18 01:41:08 PM PDT 2023

make page_owner_threshold_get() and page_owner_threshold_set() static

Reported-by: kernel test robot <lkp@intel.com>
Closes: https://lore.kernel.org/oe-kbuild-all/202305190358.IEfJNraU-lkp@intel.com/
Cc: Oscar Salvador <osalvador@suse.de>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/page_owner.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/mm/page_owner.c~mmpage_owner-filter-out-stacks-by-a-threshold-counter-fix
+++ a/mm/page_owner.c
@@ -745,13 +745,13 @@ const struct file_operations page_owner_
 
 unsigned long page_owner_stack_threshold;
 
-int page_owner_threshold_get(void *data, u64 *val)
+static int page_owner_threshold_get(void *data, u64 *val)
 {
 	*val = page_owner_stack_threshold;
 	return 0;
 }
 
-int page_owner_threshold_set(void *data, u64 val)
+static int page_owner_threshold_set(void *data, u64 val)
 {
 	page_owner_stack_threshold = val;
 	return 0;
_
